app/models/pglogical_subscription.rb

```diff
@@ -101,15 +100,24 @@ def self.provider_node_attributes(node_name)
 
   private
 
-  def node_name
-    MiqPglogical::NODE_PREFIX + MiqRegion.my_region_number.to_s
+  def update_subscription
+    disable
```

Comment:
I think this might be cleaner to wrap in a block-style method...something like `temporarily_disable` or `with_disabled` or just `disable` and the caller would do:

```ruby
temporarily_disable do
  provider_node_name = MiqPglogical.region_to_node_name(provider_region)
  pglogical.node_dsn_update(provider_node_name, dsn)
end
```

and temporarily_disable would be defined as

```ruby
def temporarily_disable
  disable
  yield
ensure
  enable
end
```